aries-site/src/examples/templates/ScreenContainer.js

```diff
@@ -4,33 +4,31 @@ import { Box, ResponsiveContext } from 'grommet';
 import { SidebarExample } from '.';
 
 export const ScreenContainer = ({ mobile, ...rest }) => (
-  <ResponsiveContext.Provider value={mobile && 'small'}>
-    <ResponsiveContext.Consumer>
-      {size => (
-        <Box
-          background="background-back"
-          width={size === 'small' ? 'medium' : '100%'}
-          height={size === 'small' ? { max: 'large' } : undefined}
-          style={{ position: 'relative' }}
-        >
-          <Box direction="row" fill>
-            {size !== 'small' && <SidebarExample />}
-            <Box
-              overflow="auto"
-              pad={{ horizontal: 'medium', bottom: 'medium' }}
-              flex
-              {...rest}
-            />
-          </Box>
-          {size === 'small' && (
-            <SidebarExample
-              style={{ position: 'absolute', bottom: 0, left: 0 }}
-            />
-          )}
+  <ResponsiveContext.Consumer>
```

Comment:
Is this file being used anywhere? Can it be deleted?

Reply:
I think it can be deleted now thanks

Reply:
File still exists? Not deleted?

Reply:
nvm. I linked to the wrong view. Disregard.